src/Auth0.php

```diff
@@ -212,7 +213,14 @@ public function exchange(
             }
 
             try {
-                $user = $this->decode($response['id_token'])->toArray();
+                $token = $this->decode($response['id_token']);
+
+                $sub = $token->getSubject() ?? '';
+                $iss = $token->getIssuer() ?? '';
+                $sid = $token->getIdentifier() ?? '';
+                $this->setBackchannel(hash('sha256', implode('|', [$sub, $iss, $sid])));
```

Comment:
I would only expect you to set the backchannel logout cache item when you get the back-channel webhook. What is happening here on login?

Reply:
This is a workaround because the PHP SDK doesn't use persistent session identifiers that we could previously tie to a backchannel request. Basically just a UUID for the session. This allows us to tie a potential incoming webhook we receive later on to this particular session.